### DISCUSSION TITLE
Welcome to NOSAD Discussions!

### DISCUSSION THREAD

**Discussion by salgo60** [salgo60/NOSAD-POC-Wikidata#1] [Announcements]
Att fundera över NOSAD <-> WIkidata <-> Wikibase prototyp

* min tanke testa findability

**Comment by jonassodergren** [comment]
Källan till varje sida på nosad är en markdown-fil. Exempelvis https://nosad.se/tips utgörs av https://gitlab.com/open-data-knowledge-sharing/wiki/-/wikis/Lista-med-delat-material. En väg att gå är att koppla NOSAD <-> Wikidata. En annan lite mer progressiv idé skulle vara att överväga om sidorna istället ska skrivas direkt i Wikidata? Själva sajten nosad.se är ju bara jag som automatiskt stylar data från WIKI. Skulle kunna testa om jag kan styla sidor från Wikidata.

**Reply by salgo60** [reply]
@jonassodergren  Vi har en Wikibase.cloud som vi kan leka med se [senaste ändringar tror du kan registrera dig där själv](https://sweopendata.wikibase.cloud/wiki/Special:RecentChanges?hidebots=1&hideWikibase=1&limit=500&days=90&urlversion=2) eller ge mig ditt email

Blir det stabilt kan vi ha saker i Wikidata....

**Reply by salgo60** [reply]
> En annan lite mer progressiv idé skulle vara att överväga om sidorna istället ska skrivas direkt i Wikidata? 

Det är så [Abstract Wikipedia](https://meta.wikimedia.org/wiki/Abstract_Wikipedia) är tänkt,,,.. --> skalar till > 300 språk... finns lite utmaningar i detta 😅

**Reply by Ainali** [reply]
@jonassodergren  Etalab i Frankrike gör så att de har sin egen katalog, och sedan så länkar de från det till Wikidataobjektet. Se exempel i botten på sidan här: https://sill.etalab.gouv.fr/software?name=VLC

Att originalpublicera på Wikidata är inte att rekommendera av två anledningar. För det första så är det tänkt att allt som finns på Wikidata ska kunna verifieras någon annanstans. Men det går ju inte om det är originalpublicerat där. För det andra så har ni inte kontroll över data där då den sköts gemensamt av Wikidatagemenskapen. Vilket innebär att det kan vara svårt att säkerställa att datamodeller och data blir på ett sätt som passar er. Det kan gå, om man "embeddar" sig i gemenskapen, men det är nog inte er huvuduppgift.

**Reply by salgo60** [reply]
@jonassodergren hör av dig om jag skall visa WIkidata och diskutera hur saker kan ge mervärde till NOSAD mitt tel: 0735152802 jag finns tillgänglig 24*7

På söndagar kör @Ainali [wikidata fika](https://sv.wikipedia.org/wiki/Wikipedia:Wikitr%C3%A4ffar#Kommande_tr%C3%A4ffar) där det dyker upp svenska gubbar som surrar Wikidata tankar vilket kan vara ett annat forum

Telegram är den plattform jag uppfattar som har mest Wikidata tryck
* "Wikidata" är allmän grupp där produkt ägare Lydia dyker upp och många andra som Lucas Werkmeister enormt smart kille på engineering
* "[Wikidata Sverige och externa kopplingar](https://t.me/joinchat/FgDj6BTRqdKkYZaCqOB_FA)" Svensk grupp låg fart men vi snackar svenska

Annars finns det massa projekt i Wikidata ex. [Wikidata:WikiProject_Sweden/Swedish_Riksdag_documents](https://www.wikidata.org/wiki/Wikidata:WikiProject_Sweden/Swedish_Riksdag_documents)

GITHUB
* exempel kör @dpriskorn 
    * [dpriskorn/svenska-vandringsleder](https://github.com/dpriskorn/svenska-vandringsleder) / [dpriskorn/hiking_trail_matcher](https://github.com/dpriskorn/hiking_trail_matcher) / [Wikidata:WikiProject_Hiking_trails](https://www.wikidata.org/wiki/Wikidata:WikiProject_Hiking_trails) där bland annat webscrapar massa amerikanska källor
    * [dpriskorn/ItemSubjector](https://github.com/dpriskorn/ItemSubjector) där ämnesord sätts i enorma mängder baserat på titel
    * [dpriskorn/LexSAOB](https://github.com/dpriskorn/LexSAOB) - SAOB identifier to Wikidata Lexemes
    
 Wikidata
 * alla sidor har diskussion och du kan göra @ping alla registrerade medlemmar 
 * finns Bybrunn där högt och lågt diskuteras
 * finns projekt som [Naturreservat](https://sv.wikipedia.org/wiki/Wikipedia:Projekt_naturreservat), svenska församlingar där lite långsiktiga saker diskuteras
* låg aktiv svensk [Wikidata frågesida](https://sv.wikipedia.org/wiki/Wikipedia:Wikidatafr%C3%A5gor)

Phabricator
* är där engineering jobbar men man kan även skapa egna ytor [phabricator.wikimedia.org/tag/wikidata](https://phabricator.wikimedia.org/tag/wikidata/)

**Reply by jonassodergren** [reply]
Tack. För https://sill.etalab.gouv.fr/software?name=VLC och liknande verkar det vara utgående länkar till kända personer, erkänd mjukvara etc. Kontaktar dig i veckan @salgo60 , vore grymt om du har lite förslag och ideér hur vi skulle kunna förstärka nosad.se.

**Reply by Ainali** [reply]
> Tack. För https://sill.etalab.gouv.fr/software?name=VLC och liknande verkar det vara utgående länkar till kända personer, erkänd mjukvara etc. 

Hmm, kanske missade jag de dåliga exemplen. Men att ha en identifierare hos er, som talar om vad det motsvarar i Wikidata är i alla fall idén. Och det gör det ju lättare för vem som helst att säkerställa vad katalogposten handlar om, oavsett vilket språk läsaren kan eller om de är en maskin. (Och all mjukvara som används i produktion av myndigheter borde klara "kändiskravet" (också känt som relevans) för att kunna ha ett Wikidataobjekt.)

**Reply by salgo60** [reply]
@jonassodergren Det jag saknar på WD är att vi utveckla ontologin.... tycker att på gott att vi "bara" hänger på Wikipedia artiklarna.... jag la upp en blog av @J

Här i Telegram gruppen funderas det just nu över vad en vandringsled är
<img width="583" alt="image" src="https://user-images.githubusercontent.com/14206509/192207779-fe094c78-918a-4272-85e0-caf4cb37a03b.png">

Gissar att @Ainali och @jonassodergren skulle kunna rita upp hur begrepp som "Public code" skall moduleras etc.... @Ainali hänger mycket mer än jag i dessa forum så du kanske har bra tankar om ontologier och hur de utvecklas.... Jan testade även ett verktyg att snyggt rita ut relationer plus att det finns 100 tals verktyg att enklare navigera/editera Wikidata se [Wikidata gadgets](https://www.wikidata.org/wiki/Wikidata:Tools)

Gissar att @gjonor och @StefanWallin har tankar... @SchermanJ gör en extremt spännande aktivitet att klassificera motioner i Riksdagen på ämnesområden se hans blogpost "[Kopplingssprint 2022](https://github.com/salgo60/NOSAD-POC-Wikidata/blob/main/Kopplingssprint%202022.md)". Ett annat projekt är det @Ainali för med @abbe98 är [govdirectory](https://github.com/govdirectory) - jag skulle vilja se att man semantiskt kopplar ihop alla goverments med "samma som" dvs. något liknande det jag tycker datastory pratar om i "[Fork me: The version-controlled political party](https://medium.com/@Lapidus/fork-me-the-version-controlled-political-party-1ab17a86188d)"

**Comment by jonassodergren** [comment]
Fattar Jan! Det var inget dåligt exempel. Kanske dåligt språk från mig, utgående i ovan sammanhang vara alltså positivt. =)

**Comment by jonassodergren** [comment]
Tycker https://gitlab.com/open-data-knowledge-sharing/publiccode.yml/-/blob/main/Kubernetes/publiccode.yml är ett bra sätt att tagga öppna repon från offentlig sektor med metadata. Mer info https://github.com/publiccodeyml/publiccode.yml

**Reply by salgo60** [reply]
@jonassodergren  Tackar för länk jag gjorde ett försök igår att skapa lite [5 star data](https://5stardata.info/en/) av era videos men lite nya fält se [ NOSAD org.csv](https://github.com/salgo60/NOSAD-POC-Wikidata/blob/main/data/NOSAD%20org.csv)

Nya kolumner
* **WikidataTalare** = om personen finns i WIkidata så är det hans Qnummer
* **TalareString** = personens namn som [strings not things](https://blog.google/products/search/introducing-knowledge-graph-things-not/)
* **Start** när videon startar på format mm:ss - funkar med Youtube rel. bra
* **GITHUB** om presentation lösning finns på GUTHUB eller om organisationen finns på GITHUB är min tanke
* **slides** länk påresentationer
* .....

<img width="1207" alt="image" src="https://user-images.githubusercontent.com/14206509/193222208-cbadafbb-1d8c-4a3a-8b19-13b67540be60.png">

## Min tro
* går att göra mycket med findability
* går att knyta ihop öppna data presentationer hos NOSAD med lagar/motioner som finns i Wikidata
* .....

## Nästa steg:
se #5
* [ ]. kanske skapa UML user case vilka intressent grupper som finns
